.github/workflows/docker-image.yml

```diff
@@ -10,6 +10,17 @@ jobs:
     steps:
       - name: ☁️ Checkout source
         uses: actions/checkout@v3
+      - name: Docker metadata action
```

Comment:
Minor and just nitpicking:

Any reason for having this as 2nd step and not after all the setup (QEMU, BuildX, Login), right before the actual docker build step?

Reply:
@reginaldbondoc Honestly just because that's where they put it in the example in the [documentation](https://github.com/marketplace/actions/docker-metadata-action) for the action. No issues changing the order though. And regarding the pr-n label, that was again just one of the tags they included in the default semver configuration. Easily modified.

Reply:
@cerrussell If you have slack please join so that we can all discuss more easily. Here is our slack https://join.slack.com/t/infisical-users/shared_invite/zt-1kdbk07ro-RtoyEt_9E~fyzGo_xQYP6g

Reply:
@maidul98 Joined.